e2e/interchaintestv8/e2esuite/grpc_query.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var queryReqToPath = make(map[string]string)
 
-func populateQueryReqToPath(ctx context.Context, chain *cosmos.CosmosChain) error {
+func PopulateQueryReqToPath(ctx context.Context, chain *cosmos.CosmosChain) error {
```

Comment:
Why public?

Reply:
🤷 don't remember, obviously not necessary. Will revert.